_posts/2021-09-02-gsoc-final-report-estefania.md

```diff
@@ -0,0 +1,299 @@
+
+# What's next?
+There is always room for improvement, and MembraneCurvature is not an exception.
+One of the main limitations of the current version of MembraneCurvature is the
+inability to calculate curvature in systems like vesicles, capsids, or micelles.
+This would definitely be a nice improvement for a future release of
+MembraneCurvature!
+
+![vesicles](https://ojeda-e.github.io/assets/images/vesicles.png)
+
+We acknowledge that scientific research would benefit from a
+tool to calculate membrane curvature in these types of systems, so we are
+considering possible approaches to include more topologies in MembraneCurvature!
+
+# Acknowledgments
+Participating in GSoC with MDAnalysis has been a unique experience. I had the
+opportunity to learn best practices in software development mentored by a group
+of incredibly talented people:
+- [@lilywang](https://github.com/lilyminium) (Lily),
+- [@IAlibay](https://github.com/IAlibay) (Irfan), 
+- [@fiona-naughton](https://github.com/fiona-naughton) (Fiona), and
+- [@orbeckst](https://github.com/orbeckst) (Oliver). 
+
+I also would like to thank [@richardjgowers](https://github.com/richardjgowers) (Richard) and 
+[@tylerjereddy](https://github.com/tylerjereddy) (Tyler) from the MDA community, who participated in our
+discussions and provided valuable insights. 
+<br> Thanks for all your valuable lessons. 
+
+[MembraneCurvature] has launched! 🚀
+
+- @ojeda-e
+
+---
+[#9]: https://github.com/MDAnalysis/membrane-curvature/pull/9
+[#34]: https://github.com/MDAnalysis/membrane-curvature/pull/34
+[#40]: https://github.com/MDAnalysis/membrane-curvature/pull/40
+[#48]: https://github.com/MDAnalysis/membrane-curvature/pull/48
+[#43]: https://github.com/MDAnalysis/membrane-curvature/pull/43
+[#44]: https://github.com/MDAnalysis/membrane-curvature/pull/44
+[#52]: https://github.com/MDAnalysis/membrane-curvature/pull/52
+[#57]: https://github.com/MDAnalysis/membrane-curvature/pull/57
+[#62]: https://github.com/MDAnalysis/membrane-curvature/pull/62
+[#64]: https://github.com/MDAnalysis/membrane-curvature/pull/64
+[#66]: https://github.com/MDAnalysis/membrane-curvature/pull/66
+[#69]: https://github.com/MDAnalysis/membrane-curvature/pull/69
+
+[`mean_curvature`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/curvature.py#L67
+[`gaussian_curvature`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/curvature.py#L39
+[`normalize_grid`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/surface.py#L108
+[`get_z_surface`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/surface.py#L52
+[`derive_surface`]: https://github.com/MDAnalysis/membrane-curvature/blob/4bb851efdeaaeb894b7c354fd019f5f6a97079de/membrane_curvature/surface.py#L22
```

Comment:
I'd make these links to your docs and I'd also add parentheses after functions, eg `mean_curvature()`.